src/tests/cli_tests.py

```diff
@@ -178,34 +178,62 @@ def rnp_params_insert_z(params, pos, z):
             params[pos:pos] = ['-z', str(z[1])]
 
 def rnp_params_insert_aead(params, pos, aead):
-    if aead:
+    if aead != None:
```

Comment:
Minor thing but the PEP8 version of this would be `if aead is not None:`. There's a lot of other things that don't align with PEP8 here though, so it's a longer-term thing, no big deal.

Reply:
Thanks, @dewyatt. Added #640 to fix this at some point.